src/fileseq/filesequence.py

```diff
@@ -1296,6 +1296,9 @@ def getPaddingNum(cls, chars, pad_style=PAD_STYLE_DEFAULT):
         if not chars:
             return 0
 
+        if chars in ['<UDIM>', '%(UDIM)d']:
```

Comment:
Looking back at this, maybe this list should be in the constants?

Reply:
It wouldn't hurt to keep the values all in one place.